src/js/controller.js

```diff
@@ -769,4 +811,14 @@ export const MediaUIRequestHandlers = {
 
     media.currentTime = seekable.end(seekable.length - 1);
   },
+  MEDIA_RENDITION_REQUEST: (media, event) => {
+    if (!media) return;
+
+    const renditionId = event.detail;
+    const selected = media.videoTracks?.[media.videoTracks?.selectedIndex ?? 0];
+
+    for (const rendition of selected?.renditions ?? []) {
+      rendition.enabled = renditionId === '' || rendition.id == renditionId;
```

Comment:
Requesting a rendition of _empty string_ feels odd. Could we make "auto" a real value?

Reply:
If this is disabling all other renditions at the same time, so only one can be enabled at a time, then I'm really not sure why we're working with arrays for _enabled_.

Reply:
the renditions API supports enabling 1, multiple or all renditions so an array makes sense.

via the UI we'll probably not support the multiple enable though...